### ISSUE TITLE
Added dired support

### ISSUE THREAD

**Issue by jtbm37** [domtronn/all-the-icons.el#24]
Thank you for your really cool emacs mode.

I just wanted to let you know that I added dired support [here](https://github.com/jtbm37/all-the-icons-dired).

**Comment by domtronn** [comment]
I like it! You should look at getting it merged as a package on Melpa. I know there's currently a package which does a similar thing called [`dired-icon.el`](https://gitlab.com/xuhdev/dired-icon), but this uses some Python Graphics library to generate it... But your version does a much better job! _(I feel)_

I was look to do this myself so thanks! :)

**Comment by jtbm37** [comment]
Thanks for the feedback.

I sent a [PR](https://github.com/melpa/melpa/pull/4397) to melpa.

I did not think there were icons for different directories. After having a look and thanks to your pointer above I managed to get this working. As you guessed I could not use it directly because I need to set the `face` and `v-adjust`.

![selection_005](https://cloud.githubusercontent.com/assets/536407/20157250/ed6fe56c-a6cb-11e6-91f5-fadbab92fad9.png)
